### PR TITLE
REST API guide audit II

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
https://jira.ez.no/browse/EZP-27855 
Review existing REST API guide:
- Extending the REST API
- JS client
- REST API reference

### CODE REVIEW

**Comment by MagdalenaZuba** [comment]
@sunpietro review ping

File: docs/api/extending_the_rest_api.md [hunk 1]
```diff
@@ -273,21 +278,24 @@ use eZ\Publish\Core\REST\Common\Message;
...
     defaults:
         _controller: myRestBundle.controller.default:sayHelloUsingPost
     methods: [POST]
 ```
+
 !!! note
-    Post Requests are not able to access the repository without performing a user authentication. More information [REST API Authentication ](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#authentication).
 
-Do not hesitate to look into the built-in InputParsers, in `eZ/Publish/Core/REST/Server/Input/Parser`, for more examples.
+    POST requests are not able to access the repository without performing a user authentication. For more information check [REST API Authentication](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#authentication).
```
Comment (DominikaK) [inline docs/api/extending_the_rest_api.md]: `Repository` with a capital ;)

File: docs/api/extending_the_rest_api.md [hunk 2]
```diff
@@ -273,21 +278,24 @@ use eZ\Publish\Core\REST\Common\Message;
...
 
+Do not hesitate to look into the built-in `InputParsers`, in `eZ/Publish/Core/REST/Server/Input/Parser`, for more examples.
```
Comment (DominikaK) [inline docs/api/extending_the_rest_api.md]: Maybe `Do not hesitate to look` -> `You can look`?

File: mkdocs.yml [hunk 3]
```diff
@@ -62,7 +62,7 @@ pages:
             - 'REST API guide': 'api/rest_api_guide.md'
             - 'REST API usage': 'api/general_rest_usage.md'
             - 'Extending the REST API': 'api/extending_the_rest_api.md'
-            - 'JS client': 'api/js_client.md'
+            - 'JavaScirpt client': 'api/js_client.md'
```
Comment (DominikaK) [inline mkdocs.yml]: Typo :)

File: docs/api/js_client.md [hunk 4]
```diff
@@ -1,42 +1,25 @@
...
+## Using the JavaScript REST API client
 
-See [below](#using-the-javascript-rest-api-client) for a guide on how to use the client, and view the [generated JS API Client reference](http://ezsystems.github.io/javascript-rest-client/).
-
-
-## Using the JavaScript REST API Client
-
-
-The JavaScript REST API Client is a JavaScript library meant to ease the use of the eZ Platform REST API. For now, it can only be used in a web browser.
+The JavaScript REST API client is a JavaScript library meant to ease the use of the eZ Platform REST API. For now, it can only be used in a web browser.
```
Comment (sunpietro) [inline docs/api/js_client.md]: Instead of `For now, it can only be` should be `It can only be`

File: docs/api/js_client.md [hunk 5]
```diff
@@ -1,42 +1,25 @@
...
 
 ## Installation
 
 ### In the PlatformUIAssetsBundle
 
-Since the JavaScript REST Client is one of the foundations of [the Platform Backend Interface](../guide/extending_ez_platform_ui.md), the client is provided by the [PlatformUIAssetsBundle](https://github.com/ezsystems/PlatformUIAssetsBundle) which is installed by default. As a result, the client is directly available and can be embedded in any Platform-generated page with the following Twig code:
+Since the JavaScript REST client is one of the foundations of [the Platform Backend Interface](../guide/extending_ez_platform_ui.md), the client is provided by the [PlatformUIAssetsBundle](https://github.com/ezsystems/PlatformUIAssetsBundle) which is installed by default. As a result, the client is directly available and can be embedded in any Platform-generated page with the following Twig code:
```
Comment (sunpietro) [inline docs/api/js_client.md]: `the Platform Backend Interface in v1` or something like this

**Pull request merged**